src/component/handlers/edit/editOnInput.js

```diff
@@ -21,7 +21,7 @@ var findAncestorOffsetKey = require('findAncestorOffsetKey');
 var nullthrows = require('nullthrows');
 
 var isGecko = UserAgent.isEngine('Gecko');
-
+var isAndroid = UserAgent.isPlatform('Android');
```

Comment:
Can we be more specific about targeting browser engines? Or does this need to apply across Android?